atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -109,6 +110,33 @@ def get_pullspecs(self, digests):
                     })
         return pullspecs
 
+    def _get_combined_logs(self):
+        return '\n'.join(self.workflow.build_logs)
+
+    def _parse_logs(self, regexp):
+        try:
+            return re.compile(regexp).search(self._get_combined_logs()).groups()
+        except:
+            return ['']
+
+    def get_error_message(self):
+        error = self._parse_logs(r'ERROR - .*plugin \'(.*)\' raised an exception: ([^(]*)')
```

Comment:
I was thinking we would *only* store the exceptions, e.g. (plugin, exception_name, exception_repr).